libs/community/langchain_community/vectorstores/sqlserver.py

```diff
@@ -127,8 +132,7 @@ def __init__(
         self._bind: Union[Connection, Engine] = (
             connection if connection else self._create_engine()
         )
-        self._embedding_store = self._get_embedding_store(table_name, db_schema)
-        self._create_table_if_not_exists()
+        self._post_connection()
```

Comment:
what does it mean to post connection here? the function is actually creating the embedding store right?

Reply:
Ah, this was me trying to see if I could reduce the number of mocks in the test to just one function. There is no specific need to have this function here. 

Yes, it calls a bunch of other functions.

Will revert back to original mode since this is unnecessary.